.github/workflows/publish-bicep.yaml

```
@@ -21,88 +21,165 @@ on:
       - main
     tags:
       - v*
-  pull_request:
-    branches:
-      - main
-  workflow_dispatch:
+  workflow_run:
+    workflows: ["Approve Publish Bicep Types"]
+    types:
+      - completed
+    workflow_dispatch:
     inputs: {}
```

Indentation is off here if I'm reading this right

```suggestion
  workflow_dispatch:
    inputs: {}
```

---

Updated